Examples/ObjectiveC/MultipleShapesExample.m

```diff
@@ -7,61 +7,80 @@ @interface MultipleShapesExample () <MGLMapViewDelegate>
 @property (nonatomic) MGLMapView *mapView;
 @end
 
+@interface MGLStyle (LayerExtensions)
+- (void)addLinesFrom:(MGLShapeSource *)source;
+- (void)addPointsFrom:(MGLShapeSource *)source;
+- (void)addPolygonsFrom:(MGLShapeSource *)source;
+@end
+
 @implementation MultipleShapesExample
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    
+
+    // Configure a map view centered on Washington, D.C.
     self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURL]];
     self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
-    self.mapView.tintColor = [UIColor darkGrayColor];
-    
     [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(38.897435, -77.039679) zoomLevel:12 animated:NO];
     self.mapView.delegate = self;
     
     [self.view addSubview:self.mapView];
 }
 
 - (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
-    
-    // Parse the GeoJSON data.
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
-        NSString *path = [[NSBundle mainBundle] pathForResource:@"metro-line" ofType:@"geojson"];
-        
-        NSData *data = [NSData dataWithContentsOfFile:path];
-        
-        dispatch_async(dispatch_get_main_queue(), ^{
-            [self drawShapeCollection:data];
-        });
-    });
+    // Since we know this file exists within this project, we'll force unwrap its value.
```

Comment:
This comment doesn't seem applicable for the Objective-C example. It may be more relevant to say something like:

 > If this data were coming from an external server, we would want to check whether the `NSURL` had a value of `nil`.